components/shared/forms/Input/Input-Forgot-Password/Input-Forgot-Password.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+type Props = {
+  type?: string;
+  placeholder: string;
+  name: string;
```

Comment:
Just in case, we put optional before required
```suggestion
  placeholder: string;
  name: string;
  type?: string;
```